src/chartLibraries/dygraph/index.js

```diff
@@ -229,10 +235,38 @@ export default (sdk, chart) => {
     }
   }
 
+  const makeSparklineOptions = () => {
```

Comment:
if we use this as a different theme, maybe is better to pass a flavour string as an attribute.

```
const flavour = chart.getAttribute("flavour") // sparkline
getOptions(flavour)
```

wdyt?